images/fedora/f28/extra-packages

```diff
@@ -1,3 +1,4 @@
+avahi
```

Comment:
It seems to me that only `nss-mdns` is necessary inside the container because it provides the GNU C Library plugin that hooks into the name resolution mechanism (or NSS) and talks to Avahi over the socket for names in the `.local` mDNS domain.

Reply:
Just for the record, we've discussed this "offline" and `nss-mdns` pulls `avahi` in.
@debarshiray mentioned he'll take care of having the package split for Fedora 34 and onwards.